File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-instructors_param.ts


import { TrainingEventInstructor_Item } from './training-event-instructor_item';
import { ISaveTrainingEventInstructors_Param } from './isave-training-event-instructors_param';

export class SaveTrainingEventInstructors_Param implements ISaveTrainingEventInstructors_Param {
  
	public TrainingEventID: number = 0;
	public Collection?: TrainingEventInstructor_Item[];
  
}

File: GTTS/INL.TrainingService.Models/SaveTrainingEventInstructors_Param.cs
using System.Collections.Generic;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventInstructors_Param : ISaveTrainingEventInstructors_Param
    {
        public long TrainingEventID { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public List<TrainingEventInstructor_Item> Collection { get; set; }
    }
}
